Pod/Classes/PINRemoteImageManager.m

```diff
@@ -163,7 +163,11 @@ - (instancetype)initWithSessionConfiguration:(NSURLSessionConfiguration *)config
         _urlSessionTaskQueue.name = @"PINRemoteImageManager Concurrent URL Session Task Queue";
         _urlSessionTaskQueue.maxConcurrentOperationCount = 10;
         
-        self.sessionManager = [[PINURLSessionManager alloc] initWithSessionConfiguration:configuration];
```

Comment:
@aciidb0mb3r To add support for custom header in PINURLSessionManager. Apparently their support for custom headers is not complete.